tribuo-tutorials/README.md

```diff
@@ -35,7 +36,311 @@ This project is compatible with Conclave v1.1. To run it, execute the following
 * Set the `conclaveRepo` property in `gradle.properties` or override it on the command line,
 so it points to your Conclave distribution.
 * Run `./gradlew host:run`, which will launch the host and wait for the client to connect.
-* Run `./gradlew client:run`, which will launch the client, connect to the host and execute all tutorials.
+* Run `./gradlew client:run --args "<productID> <codeSigner> <securityInfoSummary>"`,
```

Comment:
The order of the components doesn't actually matter, as described in `EnclaveContraint.parse`. You can just say that the arg is a constraint string as defined in the parse method.